scripts/ci/kubernetes/minikube/start_minikube.sh

```diff
@@ -41,11 +39,10 @@ case "${_UNAME_OUT}" in
 esac
 echo "Local OS is ${_MY_OS}"
 
-export MINIKUBE_WANTUPDATENOTIFICATION=false
 export MINIKUBE_WANTREPORTERRORPROMPT=false
 export CHANGE_MINIKUBE_NONE_USER=true
 
-cd $_MY_DIR
+cd ${_MY_DIR}
```

Comment:
```suggestion
cd "${_MY_DIR}"
```

while we are changing the line anyway. That or don't change the line :)

Reply:
Again shellcheck to the rescue for all other cases.

Reply:
Done